tools/db_crashtest.py

```diff
@@ -487,6 +487,8 @@ def is_direct_io_supported(dbname):
     "create_timestamped_snapshot_one_in": random.choice([0, 20]),
     # PutEntity in transactions is not yet implemented
     "use_put_entity_one_in": 0,
+    # Should not be used with TransactionDB which uses snapshot.
+    "inplace_update_support": 0,
```

Comment:
We should also disable this for Optimistic Txn DB, right?

Reply:
Added in https://github.com/facebook/rocksdb/pull/12589

Reply:
It's probably safe as long as snapshot is not used. The txn stress test here can also fail with assertions when using write prepared or write unprepared so I disabled it altogether. We can see if optimistic txn db tests would fail due to inplace_update_support.

Reply:
Ah sorry, I just saw your reply. I added it because I saw the snapshot being used here in the optimistic txn.

https://github.com/facebook/rocksdb/blob/5c1334f7633159eae1f5f89d2a7b43024b2c1916/utilities/transactions/optimistic_transaction.cc#L40-L42

`inplace_update_support` is tricky. There seem to be more features that are incompatible than compatible ones. I think we can enable the feature in a way that we start with features that we know it works together for sure instead of enabling it with all features and figuring out which one breaks together.